src/components/common/HelpText.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <header id="name-header">
+    <h2>Help with Resulting Business Type</h2>
+    <p class="mt-4">
+      A BC Limited Company and BC Community Contribution Company can amalgamate
+      but must form a BC Community Contribution Company. - BCA s. 51.98
```

Comment:
Ask Yui if he wants to use HTML entity `&sect;` (§) instead of "s.".

Reply:
"s." is good. Thanks!